apps/landingPage/src/components/Demo.js

```diff
@@ -85,6 +85,9 @@ const Container = styled.div`
 
 const HeadingWrapper = styled.div`
   padding-top: 50px;
+  @media (max-width: 600px) {
```

Comment:
We should create some media query breakpoints in a different file. Otherwise, changing that in the future could be painful.

Reply:
Agree! What values would that be? is there anything general for kiwi?

Reply:
Here's what they do in Nitro Lib
https://github.com/kiwicom/nitrolib/blob/master/src/styles/mq.js
https://github.com/kiwicom/nitrolib/blob/master/src/consts/device.js

Reply:
Thanks! Should I copy their breakpoints to the media query config then?

Reply:
Sure, why not. We could tweak them later if we need. It definitely is a good starting point.

Reply:
I also created an issue with [orbit-design-tokens](https://github.com/kiwicom/orbit-design-tokens/issues/80) for this

It would be great if it was there.

Reply:
Okay, so let's wait on `orbit-design-tokens` to figure this out then.

Reply:
Let’s use the ones from nitro-lib in the meanwhile.

Reply:
I am adding mediaQueriesConfig with nitrolib breakpoints for now